Author: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>

Crosswalk has no dependencies on GTK+, which is not even packaged on Tizen. The
original check below works fine for Tizen profiles that use Wayland, since that
implies Ozone. However, Tizen Mobile still uses X11 and we thus need an
additional check for it not to look for GTK dependencies.
--- src/build/linux/system.gyp
+++ src/build/linux/system.gyp
@@ -22,7 +22,7 @@
     'linux_link_libbrlapi%': 0,
   },
   'conditions': [
-    [ 'chromeos==0 and use_ozone==0', {
+    [ 'chromeos==0 and use_ozone==0 and tizen==0', {
       # Hide GTK and related dependencies for Chrome OS and Ozone, so they won't get
       # added back to Chrome OS and Ozone. Don't try to use GTK on Chrome OS and Ozone.
       'targets': [
